src/localization/modalStrings.js

```diff
@@ -20,7 +20,8 @@ export const modalStrings = new LocalizedStrings({
     edit_the_requisition_comment: 'Edit the requisition comment',
     edit_their_reference: 'Edit their reference',
     finalise_customer_invoice: 'Finalise will lock this invoice permanently.',
-    finalise_requisition: 'Finalise will send this requisition and lock it permanently.',
+    finalise_supplier_requisition: 'Finalise will send this requisition and lock it permanently.',
+    finalise_customer_requisition: 'Finalise will generate a finalised customer invoice, adjust inventory, and lock this requisition permanently.',
```

Comment:
Yes up to date with implementation. Doesn't exist to user, wording is user facing. When training, I think you should never mention that it exists under the hood...that's complicated. To them, we create it when they finalise the requisition.